src/lib/Scenes/Home/Components/ForYou/index.tsx

```diff
@@ -12,13 +12,15 @@ interface DataSourceRow {
   data: any
 }
 
+interface Props extends ViewProperties, RelayProps {}
```

Comment:
This is fine, so just as an FYI; you can also define this as a ‘type’, removing the need to add empty curly braces:

```ts
type Props = ViewProperties & RelayProps
```